### ISSUE TITLE
How to store data on Filecoin from a smart contract

### ISSUE THREAD

**Issue by johnnymatthews** [filecoin-project/filecoin-docs#1628]
TLDR: we can't. 

There's a longer post available here https://github.com/filecoin-project/community/discussions/601

It'd be nice to put something together explaining why we can't store directly on Filecoin completely within a blockchain-sense.

We can, _however_ create a deal and store that on the [Filecoin network using a smart contract](https://pl-strflt.notion.site/Smart-Contract-as-a-Client-Part-I-b939f665adb242528299d995793fbfe4).

## Tasks

- [ ] Explain that it's no possible to directly store data onto the network from a smart contract -- there has to be some kind of off-chain transfer of data.
- [ ] Create a tutorial based on the like above showing users how to create a storage deal.

**Comment by johnnymatthews** [comment]
This one is on hold while updates to the client contract are performed.

**Comment by johnnymatthews** [comment]
Closing in favour of https://github.com/filecoin-project/filecoin-docs/issues/1762 and https://github.com/filecoin-project/filecoin-docs/issues/1761
